subgraph/src/utils/constants/index.ts

```diff
@@ -0,0 +1,15 @@
+// Orderbook: TakeOrder(address sender, TakeOrderConfig config, uint256 input, uint256 output)
```

Comment:
could these values actually be calculated rather than hardcoded with a comment? what if the comment and hash get out of sync? how will we know?